src/test/java/org/folio/circulation/rules/Text2DroolsTest.java

```diff
@@ -495,13 +547,16 @@ public void alternatePolicyOrder() {
     }
   }
 
-  private MultiMap params(String itId, String ltId, String ptId, String slId) {
+  private MultiMap params(String itId, String ltId, String ptId, String lId) {
     MultiMap params = new CaseInsensitiveHeaders();
     params.add(ITEM_TYPE_ID_NAME, itId);
     params.add(LOAN_TYPE_ID_NAME, ltId);
     params.add(PATRON_TYPE_ID_NAME, ptId);
-    params.add(SHELVING_LOCATION_ID_NAME, slId);
+    params.add(LOCATION_ID_NAME, lId);
     return params;
   }
 
+  private Location createLocationWithId(String id){
```

Comment:
should be `private Location createLocationWithInstitution(String institutionId) {`

Reply:
I did it. But I want to add, that in the next PRs (regarding campuses and libraries) there will be other name of this method: 'createLocation(String institutionId, String campusId, String libraryId)'.